tests/DevApps/daemon-app/daemon-app-msi/Program-net60.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
It seems like a lot of duplicate code. Is there anything different?

Reply:
It's not even built. Consider it as a text file

Reply:
deleted dupe and kept it simple.